
trajectory_creator_node:
  frequency: 10.0

trajectory_sampler_node:
  frequency: 10.0

cost_grid:
  depth:
    time: 12 # discrete time steps
    x: 12 # no. of deltas for length
    y: 11 # no. of deltas for width
  delta:
    time: 0.25 #[s]
    x: 0.1 #[m]
    y: 0.05 #[m]

velocity:
  max: 0.225 #[m/s] (must be >= max(dy/dt, dx/dt))

duckietown:
  tile_size: 0.585

trajectory_sampler:
  velocity:
    max: 2.0
  k_vel: 1.0
  k_p: 4.0
  k_i: 0.0
  k_d: 0.0
  omega:
    max: 5.0
  distance:
    min: 0.05
  target_time: 0.5

obstacles:
  duckie_bot:
    radius: 0.2

actor_duckiebot_id: "duckie-0"
manual_duckiebot_id: "duckie-1"

# SIMULATION ENVIRONMENT PARAMETERS

obst-1:
  x_start: 3.0
  y_start: 0.1
  x_dim: 0.1
  y_dim: 0.1
  z_dim: 0.1
  x_end: 5.0
  y_end: 0.1
  time: 20.0
  id: 1
